x-pack/test/alerting_api_integration/security_and_spaces/group2/tests/telemetry/alerting_and_actions_telemetry.ts

```diff
@@ -198,6 +234,8 @@ export default function createAlertingAndActionsTelemetryTests({ getService }: F
       expect(telemetry.count_by_type.__index).to.equal(1);
       expect(telemetry.count_by_type['test.index-record']).to.equal(1);
       expect(telemetry.count_by_type.__webhook).to.equal(4);
+      expect(telemetry.count_by_type['__gen-ai']).to.equal(3);
+      expect(telemetry.count_gen_ai_provider_types.OpenAI).to.equal(3);
```

Comment:
@stephmilovic I figured you are probably interested in this number, so wanted to make sure you see some things might be changing ...